appveyor.yml

```
@@ -1,7 +1,7 @@
 environment:
   matrix:
   - julia_version: 1
```

For the record, you're testing twice the same platform because Julia "1" means "1.4.1", as "1.4" does.  You maybe wanted to test "1.0"?
```suggestion
  - julia_version: 1.0
```